package.json

```diff
@@ -40,7 +40,7 @@
     "@types/offscreencanvas": "^2019.6.2",
     "@types/serve-index": "^1.9.1",
     "@typescript-eslint/parser": "^4.22.0",
-    "@webgpu/types": "^0.1.7",
+    "@webgpu/types": "^0.1.8",
```

Comment:
Sorry, I caused a merge conflict in #848. Will need to change this to `0.1.8` (without the caret `^`).

Reply:
Actually, I've found that there's an error in 0.1.8. Fix incoming. Also, there's a few changes that need to be made to the CTS to roll to the new types so I think I'll handle that as a separate PR.